python/lsst/ts/cbp/component.py

```diff
@@ -163,7 +165,9 @@ def connect(self):
             self.socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
             self.socket.settimeout(10)
             self.socket.connect((self._address, self._port))
-            self.log.debug("CBP connected to {0} on port {1}".format(self._address, self._port))
+            self.log.debug(
+                "CBP connected to {0} on port {1}".format(self._address, self._port)
+            )
```

Comment:
Please consider simplifying the following lines to:
```
except TimeoutError:
    self.log.error("Socket timed out")
    raise
````
If you aren't using the exception then you don't need to capture it.